CMakeLists.txt

```diff
@@ -29,10 +29,10 @@
 #
 cmake_minimum_required(VERSION 3.5)
 cmake_policy(SET CMP0048 NEW)
-set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD 14)
 
 project(mutation++
-    VERSION 1.0.5
+    VERSION 1.0.6
```

Comment:
Hey @rubendibattista, yea I struggled a little bit thinking about this because it's not really an incompatible API change since I would argue that the previous Millikan-White classes were not really part of the public interface (though they were publicly available).  The new classes could be used independently but in reality they are just helpers to organize the data and eventually allow users to choose different models.  So in this regard I would consider it more of a new feature.  With that said, the main purpose of the change was to fix a "bug"... :)  Because of that, I just bumped the patch version, but perhaps you are right it should be at least a MINOR bump.  What do you think (based on my rambling explanation!)?  Thinking about it more, I'm inclined to agree it should be a MINOR bump.

Reply:
Well, for what I saw in the edits, the signature of some methods have changed, hence the API changed. Hence I'd say it should be a full version bump. People using the old API won't be able to use it without changing their code, wouldn't they? If that is true, it's a full version bump.

Reply:
I’m not sure I agree that there have been some method signature changes. I replaced some helper classes with new ones to improve code cohesion and reduce coupling. It’s true that if some users were using those classes in a standalone way, then yes this would break their code. However, it really didn’t make sense to use them as standalone before because they were essentially useless without the main public class OmegaVT because of the poor coupling of the old code.  Therefore I would argue that the replaced classes were never meant to be part of the public API and were only interacted with through the OmegaVT class which did not change its public API, only the private implementation. Because the new classes can now actually be used in a standalone way, they are now new features of the public API and that’s why I agree it should be a minor bump.  At a higher level I think the issue is that we don’t clearly delineate between public and private API, which is something I would like to improve in the future, but is definitely out of scope here.

Reply:
Ok, you have better visibility on this. Sounds good.